alica_engine/src/engine/AlicaEngine.cpp

```diff
@@ -104,7 +102,7 @@ bool AlicaEngine::init(AlicaCreators& creatorCtx)
     _syncModul.init();
     _variableSyncModule->init();
     _auth.init();
-    return everythingWorked;
+    return true;
```

Comment:
If this function always returns true, might as well make it return void

Reply:
Well its initializing lots of sub-components and i am sure in some cases it would make sense to return proper error codes. Maybe in a later pr we should fix that rather than changing api.